specs/commandline/plugin.md

```diff
@@ -36,15 +36,55 @@ Aliases:
   list, ls
 ```
 
+### notation plugin install
+
+```text
+Installs a plugin
+
+Usage:
+  notation plugin install [--name <plugin name>] [flags] <plugin package> 
+
+Flags:
+  -n, --name string             name of the plugin
+  -p, --plugin-package string   path to the plugin package
+  -h, --help                    help for install
+
+Aliases:
+  install, add
+```
+
+### notation plugin remove 
+
+```text
+Removes a plugin
+
+Usage:
+  notation plugin remove [flags] <plugin name>
+
+Flags:
+  -h, --help          help for remove
+
+Aliases:
+  remove, rm, uninstall, delete
+```
+
 ## Usage
 
 ### Install a plugin
 
-Currently there is no subcommand available for plugin installation. Plugin publisher should provide instructions to download and install the plugin.
+```shell
+notation plugin install --name <plugin name> --plugin-package <plugin package>
```

Comment:
Since `--name` is optional, the main usage example should be the one without the `--name` flag. 
We can add a new usage that for some users know the exact plugin name, and they want to confirm it during installation.